def div42by(divideBy):            #defines function 'div42by'
  try:                            #try to
      return 42 / divideBy        #divide 42 by the defined number assigned to 'div42by'
  except ZeroDivisionError:       #All values except any that would result in a 'ZeroDivisionError' crash
        print ('Error: You tried to divide by zero.')

print (div42by(2))        #Print result values
print (div42by(12))
print (div42by(0))        #Note that this line will display the 'Error: You tried.....' text
print (div42by(1))        #along with the value of 'None'.
                          #This is because any Functions that return without a return
                          #statement return the 'None' value.

